From 8c30f92794e44288479ada296d6d72540c11467f Mon Sep 17 00:00:00 2001
From: Justin Luth <jluth@mail.com>
Date: Sat, 22 Mar 2025 13:15:10 +0100
Subject: Revert "tdf#164201 docx convert-to: prefer Word 2010–365 Document,
 not 2007"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts 25.2.0.2 commit 858c452e5e6b7acad6932df7d1cd2f0950d47f33
on the 25.2 branch only. It is fine to remain on 25.8.

Reason for revert: There is a follow-up commit in bug 164201 that is necessary for Microsoft to be able to fully read the document. That necessary piece is only in 25.8. Making this seemingly simple change has a lot more implications than expected (primarily by Microsoft not being able to read files created according to their own specs). So I am not comfortable recommending that it be backported to a nearly-stable 25.2. Thus simply revert this optimistically-backported commit.

The specific problem that lead to the follow-up patch was the loss of create/modified date/time fields (seen only in MS Word).

The specific problem that lead to this revert is content controls only showing the placeholder text, and not the real text (seen only in MS Word).

Change-Id: I2e3d0e835282938dff86d80b17b8bb22958a0622
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/183222
Reviewed-by: Justin Luth <jluth@mail.com>
Tested-by: Jenkins
Reviewed-by: Adolfo Jayme Barrientos <fitojb@ubuntu.com>
---
 filter/source/config/fragments/types/writer_MS_Word_2007_XML.xcu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/filter/source/config/fragments/types/writer_MS_Word_2007_XML.xcu b/filter/source/config/fragments/types/writer_MS_Word_2007_XML.xcu
index e163be7cafa2..11396057a026 100644
--- a/filter/source/config/fragments/types/writer_MS_Word_2007_XML.xcu
+++ b/filter/source/config/fragments/types/writer_MS_Word_2007_XML.xcu
@@ -20,7 +20,7 @@
         <prop oor:name="URLPattern"/>
         <prop oor:name="Extensions"><value>docx</value></prop>
         <prop oor:name="MediaType"><value>application/msword</value></prop>
-        <prop oor:name="Preferred"><value>false</value></prop>
+        <prop oor:name="Preferred"><value>true</value></prop>
         <prop oor:name="PreferredFilter"><value>MS Word 2007 XML</value></prop>
         <prop oor:name="UIName"><value xml:lang="en-US">Word 2007</value></prop>
         <prop oor:name="ClipboardFormat"><value>MSWordDoc</value></prop>
-- 
cgit v1.2.3

